test/browser-pool.test.ts

```diff
@@ -600,68 +600,90 @@ describe('BrowserPool', () => {
             });
 
             describe('fingerprinting', () => {
-                test('should override fingerprint with persistentContext', async () => {
-                    const browserPoolWithFP = new BrowserPool({
-                        browserPlugins: [new PlaywrightPlugin(
+                describe.each([
+                    [
+                        'Playwright - persistent',
```

Comment:
let's move the matrix definition to a variable so we can do just `describe.each(matrix, ...)`

(haven't thought about a better name)